src/features/global/TabNotifications.js

```diff
@@ -0,0 +1,71 @@
+const Feature = require("../../core/Feature");
+const Global = require("../../sections/Global");
+const ConfigurableArray = require("../../core/ConfigurableArray");
+const Text = require("../../configurables/Text");
+const Checkbox = require("../../configurables/Checkbox");
+const Util = require("../../core/Util");
+
+class TabNotifications extends Feature {
+  constructor() {
+    super({
+      section: Global,
+      name: "Tab Notifications",
+      default: true,
+      description: "Notifications appear in browser tab.",
+      author: {
+        name: "device",
+        profile: "https://hackforums.net/member.php?action=profile&uid=1255039"
+      },
+      configurables: new ConfigurableArray(
+        new Checkbox({ id: "TNCustomSoundEnable", label: "Enable Custom Sound", default: false }),
+        new Text({ id: "TNCustomSoundLink", label: "MP3 Link", default: "https://store2.gofile.io/download/ca848df4-fc96-4496-83e3-457b77fa062c/mixkit-message-pop-alert-2354.mp3" })
+      )
+    });
+  }
+
+  run(settings) {
+    // Use an event listener to reset the page title after notifications is clicked
+    const notifyButton = $("#notifycp");
+    for (let i = 0; i < notifyButton.length; ++i) {
+      notifyButton[i].addEventListener("click", function() {
+        document.title = existingTitle;

```

Comment:
existingTitle is defined below this, so it will always be null.

You need to move `const existingTitle = ...` above `const notifyButton`.